src/Main.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import HornedBeasts from './HornedBeasts.js';
+
+class Main extends React.Component{
+    render(){
+        return(
+            <main>
+                <HornedBeasts />
+            </main>
```

Comment:
Really great job!

I am unable to clone the repo and run the app because this PR is open. (Merge after making changes...grab a TA if you run into merge conflicts)

- We are almost good here, but we need to:
- pass `title, imageUrl, and description` into each `HornedBeast` component. The first lab says you can pass whatever title, url and description that you want into each HornedBeast. 👍